packages/twenty-front/src/modules/ui/input/code-editor/components/CodeEditor.tsx

```diff
@@ -0,0 +1,77 @@
+import Editor, { Monaco, EditorProps } from '@monaco-editor/react';
+import { editor } from 'monaco-editor';
+import { codeEditorTheme } from '@/ui/input/code-editor/theme/CodeEditorTheme';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { useEffect } from 'react';
+
+export const DEFAULT_CODE = `export const handler = async (
+  event: object,
+  context: object
+): Promise<object> => {
+  // Your code here
+  return {};
+}
+`;
+
+const StyledEditor = styled(Editor)`
+  border: 1px solid ${({ theme }) => theme.border.color.medium};
+  border-top: none;
+  border-radius: 0 0 ${({ theme }) => theme.border.radius.sm}
+    ${({ theme }) => theme.border.radius.sm};
+`;
+
+type CodeEditorProps = EditorProps & { header: React.ReactNode };
+
+export const CodeEditor = ({
+  value = DEFAULT_CODE,
+  onChange,
+  language = 'typescript',
+  height = 500,
+  options = undefined,
+  header,
+}: CodeEditorProps) => {
+  const theme = useTheme();
+  const handleEditorDidMount = (
+    editor: editor.IStandaloneCodeEditor,
+    monaco: Monaco,
+  ) => {
+    monaco.editor.defineTheme('codeEditorTheme', codeEditorTheme(theme));
+    monaco.editor.setTheme('codeEditorTheme');
+  };
+  useEffect(() => {
+    const style = document.createElement('style');
+    style.innerHTML = `
+      .monaco-editor .margin .line-numbers {
+        font-weight: bold;
+      }
+    `;
+    document.head.appendChild(style);
+    return () => {
+      document.head.removeChild(style);
+    };
+  }, []);
+  return (
+    <div>
```

Comment:
you can put <> directly

Reply:
i need it to be considered as a bloc so I need a div